addons/web/static/src/core/commands/command_palette.variables.scss

```diff
@@ -0,0 +1,8 @@
+$o-command-palette-margin-top: 10vh;
+$o-command-palette-search-border: 1px solid rgba(0, 0, 0, 0.1);
+$o-command-palette-search-font-size: 1.25rem;
+$o-command-height: 2.8em;
+$o-command-palette-line-height: 1.8em;
```

Comment:
`$o-command-height`, `$o-command-palette-line-height` ... these 2 seems to just handle the entry's size and its content alignment, I'm I right?
If true just  use padding classes to increase the box-size and flex classes to handle alignment... it should be enough